OpenOversight/app/utils.py

```diff
@@ -24,6 +22,9 @@
                      Incident, Location, LicensePlate, Link, Note, Description, Salary)
 from .main.choices import RACE_CHOICES, GENDER_CHOICES
 
+# Ensure the file is read/write by the creator only
+SAVED_UMASK = os.umask(0o077)
```

Comment:
can you update this to remove the permission `o+x`? Just in case it's relevant.

Reply:
@r4v5 I thought because 7 is the last digit in the octal notation passed to `os.umask()`, other wouldn't have read, write, or execute permissions.   I'd expect `os.umask(0o077)` to cause the file to have the below permissions
`u=rwx,g=,o=`
Let me know if I'm mistaken.  Thanks.

Reply:
pretty sure @McEileen is right.
```[int10h@katahdin Desktop]$ umask 077
[int10h@katahdin Desktop]$ touch test
[int10h@katahdin Desktop]$ ls -l test
-rw-------. 1 int10h int10h 0 Aug 19 16:25 test
[int10h@katahdin Desktop]$ ```
@r4v5 can you elaborate?

Reply:
Yeah I was backwards on this